Assets/MainMenu.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MainMenu : MonoBehaviour
+{
+    public void PlayGame ()
+    {
+        SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex + 1);
```

Comment:
Cool! Looks like you can LoadScene on a scene name, not just an index from checking out the docs: https://docs.unity3d.com/ScriptReference/SceneManagement.SceneManager.LoadScene.html. 

What do you think about using the scene name instead of index?

Reply:
I did it!